src/Files.App/Filesystem/StorageEnumerators/UniversalStorageEnumerator.cs

```diff
@@ -88,9 +88,9 @@ public static class UniversalStorageEnumerator
                         else
                         {
                             var fileEntry = await AddFileAsync(item.AsBaseStorageFile(), currentStorageFolder, cancellationToken);
-                            if (fileEntry != null)
+                            if (fileEntry is not null)
                             {
-                                if (defaultIconPairs != null)
+                                if (defaultIconPairs is not null)
                                 {
                                     if (!string.IsNullOrEmpty(fileEntry.FileExtension))
                                     {
```

Comment:
Not the initial scope of the PR, but if you could take 5mn to fuse the nested `if `here, that'd be awesome!